content/en/docs/Tasks/creating-an-index.md

```diff
@@ -5,37 +5,100 @@ description: >
   Add/Remove a collection of bundles to/from an Index
 ---
 
-## Prerequisites 
+## Prerequisites
 
-- [opm](https://github.com/operator-framework/operator-registry/releases) `v1.14.0+`
-- [docker](https://docs.docker.com/install/) version `17.03`+ or [podman](https://github.com/containers/libpod/blob/master/install.md) `v1.2.0+` or [buildah](https://github.com/containers/buildah/blob/master/install.md) `v1.7+`.
+- [opm](https://github.com/operator-framework/operator-registry/releases) `v1.19.0+`
```

Comment:
This is likely fine, but we haven't cut a 1.19.x release yet - any idea what version of opm introduced file-based configs? I know there's been a lot of recent changes around that behavior, so maybe choosing a newer version would be better?

Reply:
I originally used v1.18.0, but since there are now breaking changes between 1.18 and 1.19 for DC, I thought it would be better to take the hit with potential "version does not exist" questions vs "I updated from 1.18 to 1.19 and now everything is broken" questions.

Reply:
Yeah that makes sense to me - thanks.